03-bill_splitter/bill.java

```diff
@@ -0,0 +1,24 @@
+import java.util.*;
+public class bill{
+
+	public static void main(String[] args){
+		Scanner input = new Scanner(System.in);
+		System.out.println("Welcome to the Bill-Splitter");
+		System.out.print("Enter the total, pre-tax: ");
+		double total = input.nextInt();
+
+		System.out.print("Enter tax: ");
+		double tax = input.nextInt();
```

Comment:
i was intending the tax % to be entered and you calculate the amount of tax to add!